src/Fetcher/ConfTechFetcher.php

```diff
@@ -132,6 +132,7 @@ private function denormalizeConference(array $rawConference, string $tag): ?Conf
             $query = sprintf('%s %s', $rawConference['city'], 'U.S.A.' === $rawConference['country'] ? 'United States of America' : $rawConference['country']);
             $continent = $this->locationGuesser->getContinent($query);
             $country = $this->locationGuesser->getCountry($query);
+            $coords = $this->locationGuesser->getCoordinates($query);
```

Comment:
You do an extra call/request when the contient is null, or not enabled

Reply:
BTW, I would move this block later, to avoid weird `/* @phpstan-ignore-next-line */`